server/api.js

```diff
@@ -349,6 +349,83 @@ router.get("/buyer-tender", async (req, res) => {
 		: res.status(500).send({ code: "SERVER_ERROR" });
 });
 
+router.post("/tender/:tenderId/status", async (req, res) => {
+	const tenderId = parseInt(req.params.tenderId, 10);
+	const newStatus = req.body.status;
+	const validStatuses = ["Active", "In Review", "Closed", "Awarded"];
```

Comment:
What's the difference between `Active` and `In Review`?
I think an `Awarded` tender is technically `Closed`. If we mean `Cancelled`, let's use that name instead.

Also, let's make this a global in the top of the file

Reply:
I think when they are In Review thats means maybe its past the closing date and the bids are being processed so no new bids should be able to submitted when in review. 
I will change the close to cancel